tests/test_languages.py

```diff
@@ -1910,6 +1910,11 @@ def test_exact_languages(self, datetime_string, shortnames):
         self.when_using_exact_languages()
         self.then_exact_languages_were_filtered(shortnames)
 
+    def test_none_raises_value_error(self):
+        with self.assertRaises(ValueError) as context:
+            ExactLanguages(None)
+            self.assertTrue("language cannot be None for ExactLanguages" in context.exception)
```

Comment:
@mirceachira could you change it to:

```suggestion
        with self.assertRaisesRegex(ValueError, r'language cannot be None for ExactLanguages'):
            ExactLanguages(None)
```
?

Thanks!